tests/integration_tests_control_files/time_refinement_study/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+# =======================================
+# Time Study (Linear Advection Explicit RK)
+# =======================================
+# ----------------------------------------
+# - details: [if needed]
+# ----------------------------------------
+configure_file(time_study_advection_explicit.prm time_study_advection_explicit.prm COPYONLY)
+add_test(
+  NAME 1D_Time_Study_Advection_Explicit
```

Comment:
consider making this all caps, seems to be our convention

Reply:
Sure, I will change that and follow that convention in the future.